openrouteservice/src/main/java/org/heigit/ors/api/requests/routing/RequestProfileParamsRestrictions.java

```diff
@@ -127,6 +128,14 @@
     @JsonIgnore
     private boolean hasMinWidth = false;
 
+    @ApiModelProperty(name = PARAM_SURFACE_QUALITY_KNOWN, value = "Specifies if the information on surface quality needs to be known - default false" +
```

Comment:
In relation to the general review comment - I don't think "surface quality" is the best name for this as it implies more to do with if the surface is damaged or no.

Reply:
Why is the term "quality" restricted to damage? Doesn't "quality" just mean "attribute" or "property", usually used in the context of "fitness for purpose"? So having a flat (asphalt) or rough (cobblestones) surface would also be a quality IMO.

Reply:
I agree with @sfendrich, please propose a better wording

Reply:
The quality thing and damage was what I have picked up before when working on projects related to wheelchair routing. I am just thinking that someone who does not know about the code could easily get confused as to what "surface quality" means, and from what I can work out if a single attribute is provided (e.g. surface material) then it picks it up as the edge having the surface quality as known, whereas in reality only one attribute may be known. 

If others are happy with it being referred to as "surface quality known" then OK, but in my opinion, something like "additional attributes known" would make more sense. Most people who would make use of the wheelchair routing would go via a web client anyway, and so the wording on there can be different from name of the parameter sent to the API.

Reply:
resolved as discussed on 07/14/2021 meeting https://confluence.gistools.geog.uni-heidelberg.de/display/openrouteservice/ors+backend+meeting+2021-07-14